booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,290 @@
+# Services
+
+## Get services availability
+
+Get availability for the specified service for each time unit in the specified date interval.
+
+### Request
+
+`[ApiBaseUrl]/api/distributor/v1/services/getAvailability`
+
+```json
+{
+    "Client": "My Client 1.0.0",
+    "EnterpriseId": "3edbe1b4-6739-40b7-81b3-d369d9469c48",
+    "ServiceId": "40665467-5862-4932-a861-aa6b00e2b660",
+    "StartUtc": "202-01-01T00:00:00Z",
+    "EndUtc": "2022-01-31T00:00:00Z",
+    "CategoryIds": ["4037c0ec-a59d-43f1-9d97-d6c984764e8c"],
+    "FullAmounts": false,
+    "LanguageCode": "en-GB"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Client` | string | required | Identification of the client as described in [Authorization](../guidelines/authorization.md). |
+| `EnterpriseId` | string | required | Unique identifier of the enterprise. |
+| `StartUtc` | string | required | Start date of the requested interval |
+| `EndUtc` | string | required | End date of the requested interval |
+| `CategoryIds` | array of string | optional | Unique identifiers of categories for which should be the availability computed only. If omitted, availability of all categories is returned instead. |
+| `FullAmounts` | boolean | required | Should response contain Amount objects or just Gross value.  |
+| `LanguageCode` | string | optional | Code of the language. [Supported language codes](../guidelines/supported-language-codes.md)  |
+
+
+
+### Response
+
+```json
+{
+  "TimeUnitStartsUtc": [
+    "2022-10-04T22:00:00Z",
+    "2022-10-05T22:00:00Z",
+    "2022-10-06T22:00:00Z"
+  ],
+  "CategoryAvailabilities": [
+    {
+      "CategoryId": "ddc48d9f-f21d-422a-b24a-aa6b00e2b77e",
+      "Availabilities": [
+        56,
+        40,
+        26
+      ]
+    },
+    {
+      "CategoryId": "abcdefg-08ad-4d19-98cc-aa6b00e2999e",
+      "Availabilities": [
+        0,
+        10,
+        0
+      ]
+    }
+  ],
+  "ApplicableRestrictions": []
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `TimeUnitStartsUtc` | array of timeUnit start Utc | required | Each time unit specified in request interval |
+| `CategoryAvailabilities` | array of CategoryAvailabilities | required | List of availability counts for each category. Each availability number belongs to TimeUnitStartsUtc linked by same index. |
+
+### How to read response
```

Comment:
I don't know what this section is?

Reply:
That supposted to be a suggestion how to transform the response into human readable way.
But I'll remove it for now and later I can add as a guide into different place